ui/src/pages/Game.js

```diff
@@ -66,7 +102,14 @@ class GamePage extends Component {
             <div className="row">
               <div className="col-left">
                 { this.state.ready ?
-                  <DrawCard gameId={gameId}/> :
+                  <div>
+                    <ScoreKeeper round={this.state.round}/>
+                    <DrawCard
+                      gameId={gameId}
+                      nextRound={this.startGame}
+                      nextTurn={this.nextTurn}
```

Comment:
@nneibaue  - I'm not sure this is the best way to update state in a parent component from within a child component but it seems to work for tracking the team changes and round updates... could probably do something similar for the count of unused cards? I'm basically passing the function to update the game state into the child component and they're triggered on click actions. 

I just thought I would share how I'm solving a similar problem you described with the card counter component in case it's helpful - let me know if you'd like to chat more about this or pair later! Happy to jump on a call

Reply:
So just to make sure I'm on the same page here... you are passing a callback as props to the child component, which is triggered on a button click _inside_ the child component, but the callback actually lives inside the parent component? That way the child component is actually calling the parent component's function?

Reply:
Yes, exactly!! In that way I can make my child component update state for the entire game rather than just the specific "draw card" component, as an example!! I wondered if a similar pattern might work for tracking the count of unused cards as the "card-input" and "draw-card" components change the count for the game? Then that number can be passed to another child component to show the current count?

I think if we also make sure to add the count of unused cards to the "saveState" method as well (called on component mount) then the count will also be updated on page load for other players who aren't contributing to the number change directly when they refresh their screen?